src/main/kotlin/no/nav/helse/fritakagp/integration/kafka/KafkaConfig.kt

```diff
@@ -17,6 +20,19 @@ private const val GROUP_ID_CONFIG = "helsearbeidsgiver-fritakagp"
 
 private fun envOrThrow(envVar: String) = System.getenv()[envVar] ?: throw IllegalStateException("$envVar er påkrevd miljøvariabel")
 
+fun brukernotifikasjonKafkaProps(config: ApplicationConfig) =
+    mapOf(
+        KafkaAvroSerializerConfig.SCHEMA_REGISTRY_URL_CONFIG to System.getenv("KAFKA_SCHEMA_REGISTRY"),
+        SchemaRegistryClientConfig.BASIC_AUTH_CREDENTIALS_SOURCE to "USER_INFO",
+        SchemaRegistryClientConfig.USER_INFO_CONFIG to System.getenv("KAFKA_SCHEMA_REGISTRY_USER") + ":" + System.getenv("KAFKA_SCHEMA_REGISTRY_PASSWORD"),
+        ConsumerConfig.AUTO_OFFSET_RESET_CONFIG to "earliest",
+        ConsumerConfig.CLIENT_ID_CONFIG to "fritakagp",
+        ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG to KafkaAvroSerializer::class.java,
+        ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG to KafkaAvroSerializer::class.java,
+        ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG to KafkaAvroSerializer::class.java,
+        ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG to KafkaAvroSerializer::class.java
+    ) + gcpCommonKafkaProps()
```

Comment:
Bør ikke være så mye som differerer fra gcpCommonKafkaProps.
ConsumerConfig trenger vi kanskje ikke, da vi kun produserer til denne topicen?

Reply:
Jeg kunne tenke meg å hatt helt separate configs for hvert miljø. Ikke blande og arve slik vi gjør. Hva tenker dere?

Reply:
Jeg er ikke så begeistret for sånn her:

`fun brukernotifikasjonKafkaProps(config: ApplicationConfig) =
    mapOf(
        KafkaAvroSerializerConfig.SCHEMA_REGISTRY_URL_CONFIG to System.getenv("KAFKA_SCHEMA_REGISTRY"),
        SchemaRegistryClientConfig.BASIC_AUTH_CREDENTIALS_SOURCE to "USER_INFO",
        SchemaRegistryClientConfig.USER_INFO_CONFIG to System.getenv("KAFKA_SCHEMA_REGISTRY_USER") + ":" + System.getenv("KAFKA_SCHEMA_REGISTRY_PASSWORD"),
        ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG to KafkaAvroSerializer::class.java,
        ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG to KafkaAvroSerializer::class.java
    ) + gcpCommonKafkaProps()`